scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -551,7 +558,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
       </table>
       <table class="table table-sm">
         <tr>
-          <td style="width=45%">
```

Comment:
Ah OK, didn't know that you needed ; also here! 💯

Reply:
Yes, it's also : instead of = inside of the css